runtime/jsMain/src/kotlinx/benchmark/D8EngineSupport.kt

```diff
@@ -0,0 +1,31 @@
+package kotlinx.benchmark
+
+import kotlin.time.Duration.Companion.milliseconds
+
+private external fun read(path: String): String
+private external fun write(text: String)
+
+internal object D8EngineSupport : JsEngineSupport() {
+    override fun writeFile(path: String, text: String) {
+        //WORKAROUND: D8 cannot write into files, this format will be parsed on gradle plugin side
+        write("<FILE:$path>$text<ENDFILE>")
+    }
+
+    override fun readFile(path: String): String =
+        read(path)
+
+    override fun arguments(): Array<out String> {
+        val arguments = js("globalThis.arguments.join(' ')") as String
+        return arguments.split(' ').toTypedArray()
+    }
+}
+
+internal inline fun d8MeasureTime(block: () -> Unit): Long {
+    val performance = js("(typeof self !== 'undefined' ? self : globalThis).performance")
+    val start = performance.now()
+    block()
+    val end = performance.now()
+    val startInMs = (start as Double).milliseconds.inWholeNanoseconds
+    val endInMs = (end as Double).milliseconds.inWholeNanoseconds
```

Comment:
The variables are named "..InMs" whereas their values are expressed in nanoseconds. What is supposed to be right here?